### ISSUE TITLE
use_frameworks! build error on .63.1-0

### ISSUE THREAD

**Issue by NickAVolpe** [react-native-community/react-native-tvos#83]
I am struggling to build my project after upgrading to .63.1-0 from 60.4-6. 

Is it not recommended to add `use_frameworks!` to the podfile now? 

If I add that in the RNTester project's podfile and build I will get this build error:

```
Undefined symbols for architecture arm64
Symbol: _RCTTVNavigationEventNotification
Referenced from: -[RCTTVRemoteHandler sendAppleTVEvent:toView:] in RCTTVRemoteHandler.o
```
```
Undefined symbol: _RCTShowDevMenuNotification
```

## Steps To Reproduce

1. Add use_frameworks! to RNTester/Podfile
2. pod install
3. Open RNTesterPods.xcworkspace
4. Build
5. Error occurs during build

I stumbled on this error as I am trying to resolve a different problem. I have had success building without use_frameworks! 🎉 . However I am struggling on a jenkins build and trying to determine the difference between environments. I will post a fix or comment as I work to solve it. 

![image](https://user-images.githubusercontent.com/38264728/91375737-51371680-e7d0-11ea-89b0-52c145202aab.png)

**Comment by dlowder-salesforce** [comment]
Thanks for letting me know about this, and I appreciate you working on a fix!  I'm looking into a couple of other issues now, but when I'm done with those I will look at this (if you haven't solved it by then).

**Comment by dlowder-salesforce** [comment]
@NickAVolpe I have a fix that seems to work -- not entirely happy with it, but it will at least unblock you.  See https://github.com/react-native-community/react-native-tvos/commit/8c725a7764d2e99abbd6d91e46ac56fb356e300d for the changes.

**Comment by NickAVolpe** [comment]
Right on. I was going down a similar path but got a little lost haha, thanks for showing me this. 

I ended up 'solving' the first problem I had by reverting this 1 line podspec [change for yoga](https://github.com/facebook/react-native/pull/26416). It is likely that my remote build machine has a different version of mac os and xcode.

**Comment by dlowder-salesforce** [comment]
Fixed in 0.63.1-1
